src/scripts/release.sh

```diff
@@ -23,20 +23,20 @@ fi
 
 ORIGIN_URL=$(git remote get-url origin)
 UPSTREAM="anz-bank"
-UPSTREAM_URL=$(echo "$ORIGIN_URL" | sed -n "s/\\(.*\\/\\/github.com\\/\\)\\(..*\\)\\(\\/..*.git\\)/\\1$UPSTREAM\\3/p")
+UPSTREAM_URL=$(echo "$ORIGIN_URL" | sed -n "s|\\(.*//github.com/\\)\\(..*\\)\\(/..*.git\\)|\\1$UPSTREAM\\3|p")
```

Comment:
My eyes are still bleeding, even with the slashes now non-escaped.

I suggest using single quotes and Extended Regular Expressions (EREs) with sed to make this cleaner (there's a missing escape for the dot-git, which will mean one backslash will survive, so single-quotes will stop this doubling-up):

`sed -E -n 's|(.*//github.com/)(.+)(/.+\.git)|\1'"${UPSTREAM}"'\3|p`

Maybe I'm missing something, but if $ORIGIN_URL is a single URL, you don't need `sed -n` with the `p` modifier.

Reply:
Much nicer, thanks. Done. 
(I can't possibly be held responsible for bleeding eyes).